### PR TITLE
CAMEL-13704 - Added PR template file with instructions on what to include when subm…

### PR DESCRIPTION
**Pull request by Xxyumi-hub**
CAMEL-13704 - https://issues.apache.org/jira/projects/CAMEL/issues/CAMEL-13704?filter=allopenissues
I have created a PR template file for this repo and plan to add the same template to camel-k and camel-quarkus

### CODE REVIEW

File: PR-template.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
+- [ ] Each commit in the pull request should have a meaningful subject line and body.
+- [ ] Format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
+- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
+- [ ] Run `mvn clean install` in your module to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
+- [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).
```
Comment (oscerd) [inline PR-template.html]: - the ICLA is not needed, usually it is something for contributors becoming committers or part of the PMCs.
- the point about JIRA issue is true only if the PR is large and introduce somenthing new, for little things is not needed.
- so sometimes the point 3 is not needed

File: PR-template.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
+[ ] Each commit in the pull request should have a meaningful subject line and body.
+[ ] If you're unsure, you can format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
+[ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
+[ ] Run `mvn clean install` in your module to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
```
Comment (omarsmak) [inline PR-template.html]: I'd amend this to include the sourcecheck as pretty often gets forgotten in the process:
```suggestion
[ ] Run `mvn clean install -Psourcecheck` in your module with source check enabled to make sure basic checks pass and there are no checkstyle violations. A more thorough check will be performed on your pull request automatically.
```

**Comment by Xxyumi-hub** [comment]
Can I get guidance on how to pass the failing check?

**Comment by oscerd** [comment]
It's a glitch in github action not your fault

**Comment by Xxyumi-hub** [comment]
Oh ok! Then has my change been merged to master?

**Comment by oscerd** [comment]
No. I want to wait for the other reviewers :-)

File: PR-template.html (hunk 2 above)
Comment (omarsmak) [reply PR-template.html]: @Xxyumi-hub perhaps you can take a look at this comment :)

Comment (oscerd) [reply PR-template.html]: Yes, please @Xxyumi-hub have a look at this comment and fix it.

File: PR-Template.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
+[ ] Each commit in the pull request should have a meaningful subject line and body.
+[ ] If you're unsure, you can format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
```
Comment (dmvolod) [inline PR-Template.txt]: [CAMEL-XXX]: Fixes bug in camel-file component

File: PR-Template.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
```
Comment (dmvolod) [inline PR-Template.txt]: Looks like start point is https://issues.apache.org/jira/projects/CAMEL/issues

Comment (Xxyumi-hub) [reply PR-Template.txt]: CAMEL-13704 is the one I thought I could work on, do I have to do something to claim the issue?

File: PR-Template.txt (hunk 3 above)
Comment (Xxyumi-hub) [reply PR-Template.txt]: Ah so reword my PR request like [CAMEL-13704] Added PR template file?

**Pull request merged**